server/auth/auth.go

```diff
@@ -450,6 +468,15 @@ func (s service) Type() string {
 	return "service"
 }
 
+func (s service) Raw() []knox.RawPrincipal {
+	return []knox.RawPrincipal{
+		{
+			ID:   "spiffe://" + s.domain + "/" + s.id,
+			Type: "service",
```

Comment:
Call `s.Type()` here, rather than repeating the string. That way there is a single source of truth for the value of that string.